stories/Badge.stories.tsx

```diff
@@ -61,6 +61,12 @@ export const Warning: React.FC = () => (
 export const EXTernal: React.FC = () => (
   <Badge variant="external" className="tk-mr-h">EXT</Badge>);
 
+export const BadgeSizes: React.FC = () => (
+  <>
+    <Badge size="medium" className="tk-mr-h">Badge</Badge>
+    <Badge size="small" className="tk-mr-h">Badge</Badge>
+  </>);
+
```

Comment:
Could you document the combination of medium and small with every variants? We try to cover every cases